File: Fib.go
package main

import "fmt"
import "time"

func fib(n int) int{
	if(n==0){
		return 0;
	}
	if(n==1){
		return 1;
	}
	return fib(n-1)+fib(n-2);
}
func main(){
	t:=time.Now()
	fmt.Println(fib(40))
	fmt.Println(time.Now().Sub(t))
}

File: fib.rb
#import time

def fib(n)
    if n==0
        return 0
	end
    if n==1
        return 1
	end
    return fib(n-1)+fib(n-2)
end

#t=time.time()
puts(fib(40))
#puts(time.time()-t)
